yarn.lock

```diff
@@ -0,0 +1,8973 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+"@adobe/css-tools@^4.0.1":
```

Comment:
This repo uses `npm`, not `yarn`. Please don't check in this file.

Reply:
removed